packages/material/scss/map.scss

```diff
@@ -1,14 +1,14 @@
-@import "variables";
-@import "mixins";
+@import "_variables.scss";
+@import "_mixins.scss";
 
 
 // Dependencies
-@import "common/all";
-@import "map/images/markers";
-@import "button";
-//@import "scroller";
+@import "common/_all.scss";
+@import "button.scss";
+//@import "scroller.scss";
```

Comment:
What is `scroller` and do we need to implement it?

Reply:
That needs revision for sure. Still the purpose of the PR is to switch to fully qualified names